recipes/server.rb

```diff
@@ -29,6 +30,20 @@
   notifies :restart, "service[#{node['nfs']['service']['server']}]"
 end
 
+# RHEL7 has some extra requriements per
+# https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Storage_Administration_Guide/nfs-serverconfig.html#s2-nfs-nfs-firewall-config
+if rhel? && node['platform_version'].to_f >= 7.0
+  include_recipe 'sysctl::default'
```

Comment:
@joerocklin alright its working for me, now.  It seems I had some Berkshelf related Snafu on my end.